src/bci_build/templates.py

```diff
@@ -2,13 +2,32 @@
 descriptions.
 
 """
+import datetime
 import jinja2
 
 
+INFOHEADER_TEMPLATE = f"""
+    Copyright (c) {datetime.datetime.now().year} SUSE LLC
```

Comment:
Isn't it common practice to have a starting year as well?

Reply:
the opensuse spec file formatter doesn't do it, and this basically mimics that one. I don't think the date is needed for anything, we can also omit it if thats better.